packages/web-client/src/ocs/capabilities.ts

```diff
@@ -92,6 +92,9 @@ export interface Capabilities {
           version: string
         }[]
       }
+      quickLink?: {
```

Comment:
As we follow the snake case scheme here:

```suggestion
      quick_link?: {
```